### PR TITLE
Remove warning msg when inspect containers without task

### PR DESCRIPTION
**Pull request by yldoge**
https://github.com/containerd/nerdctl/issues/2269
Removed the warning message when `nerdctl inspect` a container that doesn't have a task.

### CODE REVIEW

File: pkg/portutil/port_allocate_linux.go [hunk 1]
```diff
@@ -96,6 +99,7 @@ func portAllocate(protocol string, ip string, count uint64) (uint64, uint64, err
 			}
 		}
 		if needReturn {
+			allocateStart = int(start + count)
```
Comment (fahedouch) [inline pkg/portutil/port_allocate_linux.go:102]: is this related to container inspect warning ?

File: pkg/portutil/port_allocate_linux.go [hunk 2]
```diff
@@ -25,8 +25,11 @@ import (
 
 const (
 	// This port range is compatible with Docker, FYI https://github.com/moby/moby/blob/eb9e42a09ee123af1d95bf7d46dd738258fa2109/libnetwork/portallocator/portallocator_unix.go#L7-L12
+	allocateEnd = 60999
+)
+
+var (
 	allocateStart = 49153
```
Comment (fahedouch) [inline pkg/portutil/port_allocate_linux.go:32]:
```
const (
      allocateEnd = 60999
      allocateStart = 49153
)

File: pkg/portutil/port_allocate_linux.go (hunk 1 above)
Comment (yldoge) [reply pkg/portutil/port_allocate_linux.go:102]: No, I merged main and squashed commits.

Comment (yldoge) [reply pkg/portutil/port_allocate_linux.go:102]: > is this related to container inspect warning ?

Sorry I messed up a bit. I'll close this one and push a new.
